install_debian_autopsy_4.19.2.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
The user might have `bash` installed elsewhere. Do the following instead:
```suggestion
#!/usr/bin/env bash
```

`env` is from POSIX.1 and should be present in any system POSIX complaint, including Linux based systems.

Perhaps, if we are not using anything `bash` specific, we can use POSIX `sh`.

Reply:
The script is for debian based systems only, I assumed all of them have bash in the same place, but I can change this.